### PR TITLE
Fix a warning about 'masked_fill_ received a mask with dtype torch.ui…

### PR DESCRIPTION
**Pull request by zycv**
Many warning log while training about: "Warning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead. (masked_fill__cuda at /pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19)"

### CODE REVIEW

**Comment by by2101** [comment]
Hi, zycv,
    use .bool is really proper for pytorch >= 1.2. However, it will make some users whose pytorch is 1.1 or lower. cannot run the code. My solution is to use "ignore warning" argument of python. Do you have any good solution?

**Comment by zycv** [comment]
yes, it seems that your method will be more compatible, please close this PR！
